File: Assignment13/program2.c
//2. Print all even numbers upto N

#include <stdio.h>

void print_even_numbers ( int limit)
{
int iCnt = 0;
for(iCnt = 1; iCnt <= limit; iCnt++)
{
    if(iCnt % 2 == 0)
    {
        printf("%d\t", iCnt);
    }
}
}
//Time Complexity: O(N)

int main()
{
    int limit;

    printf("Enter Number:  ");
    scanf("%d", &limit);
    print_even_numbers(limit);
    return 0;
}

File: Assignment18/Program2.java
/* Write a program to print all even numbers upto N */
package Assignment18;

class Logic
{
    void printEvenNumbers(int n)
    {
        int iCnt = 0;
        for(iCnt = 0 ; iCnt<= n ; iCnt++)
        {
            if(iCnt%2 == 0)
            {
                System.out.println(iCnt);
            }
        }
    }
}
class Program2 
{
    public static void main(String A[]) 
    {
        Logic obj = new Logic();
        obj.printEvenNumbers(20);    
    }
}
